source/draw/RasterCanvas.ooc

```diff
@@ -71,4 +71,17 @@ RasterCanvas: abstract class extends Canvas {
 	_map: func (point: IntPoint2D) -> IntPoint2D {
 		point + this _size / 2
 	}
+	resizeNearestNeighbour: static func <T> (sourceBuffer, resultBuffer: T*, sourceSize, resultSize: IntVector2D, sourceStride, resultStride, bytesPerPixel: Int) {
```

Comment:
This method is just moved from `RasterMonochrome` and parametrized with pixel type.